source/staking/test/unit.js

```diff
@@ -326,44 +214,30 @@ describe('Staking Unit', () => {
         await ethers.provider.send('evm_mine')
```

Comment:
I know you didn't add this in, but I was looking and this is a really weird way to increase time...we're making a 100 `evm_mine` calls...this particular test is much slower than the others...you can change it to the following and it works faster:

`    await ethers.provider.send('evm_increaseTime', [10])`
`    await ethers.provider.send('evm_mine')`

Reply:
Done